packages/api/endpoints/rules.js

```diff
@@ -170,7 +175,7 @@ async function del(req, res) {
 
   await dbClient.transaction(async (trx) => {
     await trx(tableNames.rules).where({ name }).del();
```

Comment:
Same here: can we use/add a rules model `delete`? Really we should probably have a `delete` method on the base model

Reply:
That is a change that needs made, but it's outside of the scope of a 'fix the broken upset logic' ticket.     I'm happy to do prioritize/ do it as I agree we shouldn't leave it,  but I think I'd prefer to file a separate PR to address the db model/endpoint inconsistencies as that's not the primary focus of this ticket.